src/wp-includes/class-wp-autoload.php

```diff
@@ -0,0 +1,500 @@
+<?php
+/**
+ * Autoloader for WordPress classes.
+ *
+ * Include this file if you'd like to avoid having to create your own autoloader.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Autoloader.
+ */
+final class WP_Autoload {
+
+	/**
+	 * An array of classes and their respective paths.
+	 *
+	 * Note: The classnames should be lowercased.
+	 *
+	 * @access private
+	 *
+	 * @var array
+	 */
+	const CLASSES_PATHS = array(
+		/* Classes in the wp-includes/ folder. */
```

Comment:
Coding standards for [one line comments](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-inline-comments) are to use `//` 

Applies in a few places so I won't repeat myself.

```suggestion
		// Classes in the wp-includes/ folder.
```